src/Compiler/CompilerUtils.cpp

```diff
@@ -40,19 +40,22 @@ using namespace mlir;
 using namespace onnx_mlir;
 
 const std::string OnnxMlirEnvOptionName = "ONNX_MLIR_FLAGS";
-static const std::string OnnxMlirVersion = "onnx-mlir version 1.0.0";
-static const std::string OnnxMlirVersionString =
+
+namespace {
+
+static std::string getOnnxMlirFullVersion() {
```

Comment:
This should move below with the other static functions

Reply:
sure, done.